app/web/features/dashboard/CommunitiesSection.tsx

```diff
@@ -0,0 +1,70 @@
+import { Link as MuiLink, makeStyles, Typography } from "@material-ui/core";
+import CommunitiesDialog from "features/dashboard/CommunitiesDialog";
+import CommunitiesList from "features/dashboard/CommunitiesList";
+import { Trans, useTranslation } from "i18n";
+import { DASHBOARD, GLOBAL } from "i18n/namespaces";
+import { useState } from "react";
+
+const COMMUNITY_BUILDER_FORM_LINK =
+  "https://couchers.org/community-builder-form";
+
+const useStyles = makeStyles((theme) => ({
+  createCommunityText: {
+    paddingBlockStart: theme.spacing(2),
+  },
+}));
+
+export default function CommunitiesSection() {
+  const { t } = useTranslation([GLOBAL, DASHBOARD]);
+  const classes = useStyles();
+  const [isCommunitiesDialogOpen, setIsCommunitiesDialogOpen] =
+    useState<boolean>(false);
```

Comment:
```suggestion
    useState(false);
```
Type inference is enough here and we prefer to lean on that as much as possible to write less code

Reply:
Sounds good, I'll do that moving forward here.